### PR TITLE
vite 4.4.8

### PR DESCRIPTION
**Pull request by dnhn**
Created by `brew bump`

-----

Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/vite.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@
 class Vite < Formula
   desc "Next generation frontend tooling. It's fast!"
   homepage "https://vitejs.dev/"
-  url "https://registry.npmjs.org/vite/-/vite-4.4.7.tgz"
```
Comment (chenrui333) [inline Formula/vite.rb:6]: what is the rationale for this change?

Comment (dnhn) [reply Formula/vite.rb:6]: This was my mistake, I’ll revert this to the previous URL format when the NPM package is published.

Comment (chenrui333) [reply Formula/vite.rb:6]: 👍

**Comment by chenrui333** [comment]
let's close this for now (feel free to reopen/submit a new PR later)
